src/vs/base/browser/ui/tree/asyncDataTree.ts

```diff
@@ -676,12 +676,12 @@ export class AsyncDataTree<TInput, T, TFilterData = void> implements IDisposable
 		this.tree.setFocus(nodes, browserEvent);
 	}
 
-	focusNext(n = 1, loop = false, browserEvent?: UIEvent): void {
-		this.tree.focusNext(n, loop, browserEvent);
+	focusNext(n = 1, loop = false, browserEvent?: UIEvent, filter?: (node: any) => boolean): void {
```

Comment:
I tried to avoid using the `any` type here and in three subsequent occurrences but couldn't work out a better one. A TS expert would probably know what to do.